flang/lib/Frontend/CompilerInvocation.cpp

```diff
@@ -832,12 +833,15 @@ static bool parseSemaArgs(CompilerInvocation &res, llvm::opt::ArgList &args,
   // -J/module-dir option
   auto moduleDirList =
       args.getAllArgValues(clang::driver::options::OPT_module_dir);
-  // User can only specify -J/-module-dir once
+  // User can only specify one -J/-module-dir directory
```

Comment:
"...but may repeat -J as long as the directory is the same each time."

Reply:
Yeah, that sounds better, thanks.